build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
> Doing the crossgen-ing on Non-Windows in Helix, in which case, it would have to happen N times, once for each TestWrapper .dll, since we send those N jobs from a Windows machine.

We build CoreCLR for each platform on that platform. Is it not possible to restore the CorEFX packages, which need to be in Core_ROOT sent to Helix, to get restored as part of the product build and thus, you will get a chance to precompile them before they get sent?